vantage6-server/vantage6/server/resource/user.py

```diff
@@ -870,19 +909,27 @@ def delete(self, id):
                     "automatically together with this user."
                 }, HTTPStatus.BAD_REQUEST
             else:
-                log.warn(
-                    f"Deleting {len(user.created_tasks)} tasks created by"
-                    f" user id={id}"
+                log.warning(
+                    "Deleting %s tasks created by user id=%s",
+                    len(user.created_tasks),
+                    id,
                 )
                 for task in user.created_tasks:
                     task.delete()
 
-        self._delete_user_in_keycloak(user)
+        if params.get("delete_from_keycloak", True):
+            self._delete_user_in_keycloak(user)
+        else:
+            log.warning("User id=%s will not be deleted from Keycloak", id)
```

Comment:
I think this level is too high, they where already quite explicit when setting the flag `delete_from_keycloak`.
```suggestion
            log.info("User id=%s will not be deleted from Keycloak", id)
```